libraries/AP_HAL_ChibiOS/hwdef/3DRControlN1/README.md

```diff
@@ -0,0 +1,215 @@
+| I2C   | I2C2_SDA  | 23  | 24  | RESERVED  | -     |
+| I2C   | I2C2_SCL  | 25  | 26  | GND       | -     |
+| POWER | GND       | 27  | 28  | ADC2      | ADC   |
+| I2C   | I2C1_SDA  | 29  | 30  | ADC1      | ADC   |
+| I2C   | I2C1_SCL  | 31  | 32  | GND       | POWER |
+| POWER | GND       | 33  | 34  | RESERVED  | -     |
+| SG7   | UART5_RX  | 35  | 36  | RESERVED  | -     |
+| SG7   | UART5_TX  | 37  | 38  | RESERVED  | -     |
+| POWER | GND       | 39  | 40  | RESERVED  | -     |
+| SG3   | UART7_RTS | 41  | 42  | GND       | POWER |
+| SG3   | UART7_CTS | 43  | 44  | GP4       | GPIO  |
+| SG3   | UART7_RX  | 45  | 46  | GP3       | GPIO  |
+| SG3   | UART7_TX  | 47  | 48  | GP2       | GPIO  |
+| -     | RESERVED  | 49  | 50  | GP1       | GPIO  |
+| -     | RESERVED  | 51  | 52  | GND       | POWER |
+| POWER | GND       | 53  | 54  | CH12      | GPIO  |
+| -     | RESERVED  | 55  | 56  | CH11      | GPIO  |
+| POWER | GND       | 57  | 58  | CH10      | GPIO  |
+| -     | RESERVED  | 59  | 60  | CH9       | GPIO  |
+| -     | RESERVED  | 61  | 62  | GND       | POWER |
+| -     | RESERVED  | 63  | 64  | CH8       | GPIO  |
+| POWER | GND       | 65  | 66  | CH7       | GPIO  |
+| -     | RESERVED  | 67  | 68  | CH6       | GPIO  |
+| POWER | GND       | 69  | 70  | CH5       | GPIO  |
+| -     | RESERVED  | 71  | 72  | GND       | POWER |
+| -     | RESERVED  | 73  | 74  | CH4       | GPIO  |
+| POWER | GND       | 75  | 76  | CH3       | GPIO  |
+| -     | RESERVED  | 77  | 78  | CH2       | GPIO  |
+| -     | RESERVED  | 79  | 80  | CH1       | GPIO  |
+
+
+## UART Mapping
+
+- SERIAL0 -> USB1, for GCS connection
+
+- SERIAL1 -> USART2 (TELEM 1)
+
+- SERIAL2 -> UART4 (TELEM 2)
+
+- SERIAL3 -> UART8 (GPS)
+
+- SERIAL4 -> USART3 (GPS 2)
+
+- SERIAL5 -> USART6
+
+- SERIAL6 -> UART7
+
+- SERIAL7 -> USART1
+
+## Analog Inputs
+
+The Control N1 has 2 user ADC inputs:
+
+- ADC1 Pin18 -> Battery Voltage
+- ADC1 Pin10 -> Battery Current
+
+And one internal ADC input:
+
+- ADC1 Pin8 -> Vcc (5V)
+
+## PWM Output
+
+The Control N1 supports 12 PWM outputs. All DShot and BiDirDShot capable (TBC)
+
+The PWM outputs are distributed in 3 groups:
+
+- PWM 1-4 in group 1
+- PWM 5-8 in group 2
+- PWM 9-12 in group 4
+
+Channels within the same group must use only one output rate. If any channel is using DShot or BiDirDShot the rest of the group will use the said output type.
+
+## Addressable LED
+
+The RGB LED onboard is a WS2812B addressable type LED, connected to PWM channel 13 (TIM3_CH4).
+The LED's data out pin is exposed as ADDR_LED.
+
+## Power Supply
+
+This board requires a 5V, 500mA power supply.
+
+## Battery Monitoring
+
+Analog voltage and current or I2C sensors may be used. The following settings need to be set to work with a Power Zero Module (M10077):
+
+- BATT_MONITOR 4
+- BATT_VOLT_PIN 18
+- BATT_CURR_PIN 10
+- BATT_VOLT_SCALE 15.3
+- BATT_CURR_SCALE 50.0
+
+_Other Power Modules need to be adjusted accordingly_
+
+## Build
```

Comment:
not required, but line like this one:
F4BY_H743 can be found here <https://firmware.ardupilot.org> in sub-folders labeled “F4BY_H743”.